TERMSOFUSE.txt

```diff
@@ -0,0 +1,14 @@
+Terms of Use
```

Comment:
This is a `.txt` file. Should it be `.md`?

Reply:
The original file was in txt and I did not feel like adding in the additional formatting that md should have. I don't feel strongly about either format so I am leaning towards the path of least effort.

Reply:
Roger that.